Makefile

```diff
@@ -23,7 +23,7 @@ target/bundle/index.html:
 
 target/bundle/main.js: target/public/cljs-out/prod-main.js
```

Comment:
You changed where it attempts to copy from, but you still need to update the `target/public/cljs-out/prod-main.js` target to match your webpack build changes

Reply:
this looks like just merge from your other one